stdlib/public/SDK/Foundation/Data.swift

```diff
@@ -198,7 +293,7 @@ public final class _DataStorage {
             d.enumerateBytes { (ptr, range, stop) in
                 var stopv = false
                 let bytePtr = ptr.bindMemory(to: UInt8.self, capacity: range.length)
-                block(UnsafeBufferPointer(start: bytePtr, count: range.length), range.location, &stopv)
+                block(UnsafeBufferPointer(start: bytePtr, count: range.length), range.location - _offset, &stopv)
```

Comment:
As I mentioned to you separately, these still need to be clamped to the `range` parameter passed to `enumerateBytes`. It might be a good idea to avoid the variable aliasing as well.

Reply:
yep, working on a test and patch for that one; thanks again